app/Http/Controllers/BiosController.php

```diff
@@ -0,0 +1,144 @@
+<?php namespace Symposium\Http\Controllers;
+
+use Symposium\Services\CreateBioForm;
+use Auth;
+use Bio;
+use Illuminate\Support\Collection;
+use Input;
+use Log;
+use Redirect;
+use Session;
+use Validator;
+use View;
+
+use Symposium\Exceptions\ValidationException;
+
+class BiosController extends BaseController
+{
+    // @todo: Why is this here but only used for update? Why are they not using the same pattern?
+    protected $validation = [
+    ];
+
+    public function index()
+    {
+        $bios = Bio::where('user_id', Auth::user()->id)
+            ->orderBy('nickname')
+            ->get();
+
+        return View::make('bios.index')
+            ->with('bios', $bios);
+    }
+
+    public function create()
+    {
+        return View::make('bios.create')
+            ->with('bio', new Bio());
+    }
+
+    public function store()
+    {
+        $form = CreateBioForm::fillOut(Input::all(), Auth::user());
+
+        try {
+            $bio = $form->complete();
+        } catch (ValidationException $e) {
+            return Redirect::to('bios/create')
+                ->withErrors($e->errors())
+                ->withInput();
+        }
+
+        Session::flash('message', 'Successfully created new bio.');
+
+        return Redirect::to('/bios/' . $bio->id);
```

Comment:
We should use named routes for this stuff, the `resource` routing style still gives you route names. This one would be `bios.show`.